key-manager/src/main/scala/org/bitcoins/keymanager/WalletStorage.scala

```diff
@@ -174,20 +174,17 @@ object WalletStorage extends KeyManagerLogger {
     }
   }
 
-  /** Reads the raw encrypted mnemonic from disk,
+  /** Reads the raw encrypted mnemonic from json,
     * performing no decryption
     */
-  private def readEncryptedMnemonicFromDisk(
-      seedPath: Path): Either[ReadMnemonicError, EncryptedSeed] = {
-
-    val jsonE = readJsonFromDisk(seedPath)
-
+  private def readEncryptedMnemonicFromJson(
+      json: Value): Either[ReadMnemonicError, EncryptedSeed] = {
     import MnemonicJsonKeys._
     import ReadMnemonicError._
 
     val readJsonTupleEither: Either[
       ReadMnemonicError,
-      (String, String, String, Long)] = jsonE.flatMap { json =>
+      (String, String, String, Long)] = {
```

Comment:
perhaps we should make a type for this?

Reply:
eh, we could but it'd only be used here. I don't think it's a big deal

Reply:
It makes code reviewers and maintainers lives a lot easier to be able to access things like `.rawIv`, `.rawCipherText` etc rather than having to do `._1` `._2` `._4` etc. It's very hard to remember the ordering of tuples, _especially_ when most of them are of the same type (`String` in this case).